src/app/admin/components/create-card/create-card.component.html

```diff
@@ -0,0 +1,44 @@
+<div class="card">
+  <h4 class="title">Create new card</h4>
+
+  <form class="card-form"
+  #cardForm="ngForm"
+  (ngSubmit)="onSubmit(cardForm)">
+    <div class="form-group">
+      <label for="title">Title:</label>
+      <input id="title" type="text"
+      [(ngModel)]="card.title"
+      name="title"
+      required
+      >
+    </div>
```

Comment:
[optional]
here you can watch how to create custom form controls, and remove this x4 duplicated peace of code.